sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -1,111 +1,64 @@
+# Prerequisites
 
-### Get the required tools:
+* `git` - https://git-scm.com/downloads
+* `cmake` version 3.2 or later - http://www.cmake.org/download/
+* `python` - https://www.python.org/downloads/release/python-2716/
+* C++ compiler
+  * Linux: `GCC` version 5.1.0 or later (including libstdc++) -
+    https://gcc.gnu.org/install/
+  * Windows: `Visual Studio` version 15.7 preview 4 or later -
+    https://visualstudio.microsoft.com/downloads/
 
-   a. `git` - for downloading the sources (Get it at https://git-scm.com/downloads)
+## Create SYCL workspace
 
-   b. `cmake` - for building the compiler and tools, version 3.2 or later (Get it at: http://www.cmake.org/download)
+Throughout this document `SYCL_HOME` denotes the path to the local directory
+created as SYCL workspace. It might be useful to create an environment variable
+with the same name.
 
-   c. `python` - for building the compiler and running tests (Get it at: https://www.python.org/downloads/release/python-2716/ )
+**Linux**
 
-   d. `Visual Studio 2017 or later` (Windows only. Get it at: https://visualstudio.microsoft.com/downloads/)
-
-
-# Configure environment:
-For simplicity it is assumed below that the environment variable SYCL_HOME contains path to a folder where the SYCL compiler and runtime will be stored.
-### Linux:
 ```bash
-export SYCL_HOME=/export/home/workspaces/sycl_workspace
+export SYCL_HOME=/export/home/sycl_workspace
 mkdir $SYCL_HOME
 ```
-### Windows:
-Open a developer command prompt using one of tho methods:
-- Click start menu and search for the command prompt. So, for MSVC-2017 it is '`x64 Native Tools Command Prompt for VS 2017`'
-- run 'cmd' and then '`"C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" x64`'
 
-```bash
-set SYCL_HOME=%USERPROFILE%\workspaces\sycl_workspace
-mkdir %SYCL_HOME%
-```
-# Get `OpenCL-Headers` and `OpenCL-ICD-Loader`
-
-**These 2 steps are optional.** The Compiler build process is going to look for available OpenCL SDK on the local machine.
-If it finds suitable OpenCL, it will reuse it. Otherwise, it will automatically download `OpenCL-Headers` and `OpenCL-ICD-Loader` from GitHub and build it.
-You may want to run these steps if have some unexpected problems caused by `OpenCL-Headers` or `OpenCL-ICD-Loader` at the compiler build phase.
-
-## Get `OpenCL-Headers`
-### Linux:
-```bash
-cd $SYCL_HOME
-git clone https://github.com/KhronosGroup/OpenCL-Headers
-export OPENCL_HEADERS=$SYCL_HOME/OpenCL-Headers
-```
-
- ### Windows:
-```bash
-cd %SYCL_HOME%
-git clone https://github.com/KhronosGroup/OpenCL-Headers
-set OPENCL_HEADERS=%SYCL_HOME%\OpenCL-Headers
-```
+**Windows**
 
-## Get `OpenCL-ICD-Loader`
-You can also find the most recent instructions for this component at https://github.com/KhronosGroup/OpenCL-ICD-Loader
+Open a developer command prompt using one of two methods:
 
-### Linux:
-```bash
-cd $SYCL_HOME
-git clone https://github.com/KhronosGroup/OpenCL-ICD-Loader
```

Comment:
Alexey, I missed the fact that you removed the build steps for ICD-Loader.
I believe that was not good move. make and nmake generators are stable, I also had some success with ninja during the main SYCL compiler build, but when ICD-Loader and Headers are not pre-built before the main SYCL build, ninja again gives this error:

D:\iusers\vklochko\ws\gs_public\build_ninja2>ninja sycl-toolchain
ninja: error: 'lib/OpenCL.lib', needed by 'bin/sycl.dll', missing and no known rule to make it

Reply:
@v-klochkov, I encountered similar error (unfortunately, I've lost build logs) while using MSVC generator: we definitely have something misconfigured/missing

Reply:
This looks like a bug in the cmake script, feel free to file a bug report!